[2025-02-19T11:39:45.341+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T11:39:45.351+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.save_data manual__2025-02-19T11:38:32.325643+00:00 [queued]>
[2025-02-19T11:39:45.446+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.save_data manual__2025-02-19T11:38:32.325643+00:00 [queued]>
[2025-02-19T11:39:45.447+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-19T11:39:45.453+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): save_data> on 2025-02-19 11:38:32.325643+00:00
[2025-02-19T11:39:45.457+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=113) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T11:39:45.458+0000] {standard_task_runner.py:72} INFO - Started process 115 to run task
[2025-02-19T11:39:45.459+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'save_data', 'manual__2025-02-19T11:38:32.325643+00:00', '--job-id', '912', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpk27ro9en']
[2025-02-19T11:39:45.459+0000] {standard_task_runner.py:105} INFO - Job 912: Subtask save_data
[2025-02-19T11:39:45.486+0000] {task_command.py:467} INFO - Running <TaskInstance: pipeline.save_data manual__2025-02-19T11:38:32.325643+00:00 [running]> on host 23903a4a7553
[2025-02-19T11:39:45.543+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T11:38:32.325643+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-19T11:38:32.325643+00:00'
[2025-02-19T11:39:45.544+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-19T11:39:45.544+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-19T11:39:45.544+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2025-02-19 11:39:45.351740+00:00
[2025-02-19T11:39:45.545+0000] {logging_mixin.py:190} INFO - Dag name:pipeline and current dag run status:running
[2025-02-19T11:39:45.545+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T11:39:45.545+0000] {safe_data.py:23} INFO - Starting to save data with 0 records.
[2025-02-19T11:39:45.545+0000] {safe_data.py:31} INFO - File temp_file_for_data.txt exists. Attempting to read existing data.
[2025-02-19T11:39:45.545+0000] {safe_data.py:36} INFO - Successfully read 2 existing records from temp_file_for_data.txt.
[2025-02-19T11:39:45.547+0000] {safe_data.py:47} INFO - Added 0 new records to the existing data.
[2025-02-19T11:39:45.547+0000] {safe_data.py:52} INFO - Successfully saved 2 records to temp_file_for_data.txt.
[2025-02-19T11:39:45.547+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-19T11:39:45.551+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T11:39:45.552+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=save_data, run_id=manual__2025-02-19T11:38:32.325643+00:00, execution_date=20250219T113832, start_date=20250219T113945, end_date=20250219T113945
[2025-02-19T11:39:45.561+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-19T11:39:45.561+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-19T11:39:45.561+0000] {logging_mixin.py:190} INFO - Dag name:pipeline queued_at:2025-02-19 11:38:32.343662+00:00
[2025-02-19T11:39:45.562+0000] {logging_mixin.py:190} INFO - Task hostname:23903a4a7553 operator:_PythonDecoratedOperator
[2025-02-19T11:39:45.600+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T11:39:45.609+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T11:39:45.610+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
